tests/flow/tests_pytorch.py

```diff
@@ -34,7 +34,7 @@ def test_pytorch_chunked_modelset(env):
     env.assertEqual(len(model2), len(model_chunks))
     env.assertTrue(all([el1 == el2 for el1, el2 in zip(model2, model_chunks)]))
 
-    model3 = con.execute_command('AI.MODELGET', 'm2', 'META', 'BLOB')[-1]
+    model3 = con.execute_command('AI.MODELGET', 'm2', 'META', 'BLOB')[15]  # Extract the BLOB list from the result
```

Comment:
why this was changed?

Reply:
Because the blob position hasn't changed, while the list has extended (by adding the `minbatchtimeout` at the end). So the blob(s) are not in last position anymore...